### PR TITLE
archs38: set device vendor and model variables

### PR DESCRIPTION
**Pull request by mwarning**
This fixes the profiles.json output.

Signed-off-by: Moritz Warning <moritzwarning@web.de>

### CODE REVIEW

**Comment by mwarning** [comment]
Note: This has no effect on the names of any produced images.

**Comment by mwarning** [comment]
This is related to https://github.com/openwrt/openwrt/pull/3973

**Comment by mwarning** [comment]
@adschm care for a review?

File: target/linux/archs38/image/Makefile [hunk 1]
```diff
@@ -24,6 +24,9 @@ endef
 
 define Device/nsim_hs
 	$(call Device/vmlinux)
+	DEVICE_VENDOR := Synopsys
+	DEVICE_MODEL := SDP board
+	DEVICE_VARIANT := AXS103
```
Comment (adschm) [inline target/linux/archs38/image/Makefile:29]: axs103, nsim_hs_idu, ...?

**Comment by hauke** [comment]
Thank you for the patch. I applied it to master.

**Pull request closed without merging**